grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -717,6 +771,48 @@ static GrpcStatusFunction toGrpcStatusFunction(
         return interceptors;
     }
 
+    private void collectDecorators(Class<?> clazz,
+                                   @Nullable String path,
+                                   ServerServiceDefinition serverServiceDefinition) {
+        final String serviceName =
+                path == null ? serverServiceDefinition.getServiceDescriptor().getName() : path;
+        final List<DecoratorAndOrder> serviceDecorators = DecoratorUtil.collectDecorators(clazz);
+        if (!serviceDecorators.isEmpty()) {
+            methodDecorators.put('/' + serviceName, DecoratorUtil.collectDecorators(clazz));
+        }
+        // In gRPC, A method name is unique.
+        final Map<String, Method> methods = DecoratorUtil.decoratorMethods(clazz)
+                                                         .stream()
+                                                         .collect(Collectors.toMap(Method::getName,
+                                                                                   Function.identity()));
```

Comment:
Should we remove serviceDecorators and check all public methods to create `methodDecorators`?
```suggestion
        // In gRPC, a method name is unique.
        final Map<String, Method> methods = new HashMap<>();
        for (Method method: getAllMethods(clazz, withModifier(Modifier.PUBLIC))) {
            methods.put(method.getName(), method);
        }
```

Reply:
I understand it.
Fixed it -> acc8668